chevah/compat
# [#5267] Fix str vs unicode testing. Better text generation.

Scope
=====

This contains a bunch of small fixed found while working on chevah/server


Changes
=======

* Fix Twisted test case cleanup when a test fails
* Fix testing that when 2 values are equal, if they are string, they also have the same type.
* Update text generation to include uppercase letters.


How to try and test the changes
===============================

reviewers: @dumol 

This is more like a FYI.

@dumol  I think that we should enable OpenBSD 3.6 for compat to match your system.

With this branch, I hope that we can have a green build on all slaves.

For my test, the new `set_download_commands` from `paver.sh` is too verbose :)
I don't need to be reminded each time, I run `paver.sh`... maybe only show the output for `paver deps` or when curl/wget is really used.  
For example `paver.sh remote` doesn't uses wget/curl
